src/orders/components/OrderReturnPage/OrderReturnRefundItemsCard/ReturnItemsCard.tsx

```diff
@@ -0,0 +1,234 @@
+        paddingRight: 0
+      },
+      remainingQuantity: {
+        ...inputPadding,
+        color: theme.palette.text.secondary,
+        whiteSpace: "nowrap"
+      },
+      setMaximalQuantityButton: {
+        marginBottom: theme.spacing(1),
+        marginTop: theme.spacing(2),
+        padding: 0
+      }
+    };
+  },
+  { name: "ItemsCard" }
+);
+
+const messages = defineMessages({
+  improperValue: {
+    defaultMessage: "Improper value",
+    description: "error message"
+  },
+  titleFulfilled: {
+    defaultMessage: "Fulfillment - #{fulfilmentId}",
+    description: "section header"
+  },
+  titleUnfulfilled: {
+    defaultMessage: "Unfulfilled Items",
+    description: "section header"
+  }
+});
+
+interface OrderReturnRefundLinesCardProps {
+  onChangeQuantity: FormsetChange<number>;
+  fulfilmentId?: string;
+  canReplace?: boolean;
+  lines: OrderDetails_order_lines[];
+  order: OrderDetails_order;
+  itemsSelections: FormsetReplacementData;
+  itemsQuantities: FormsetQuantityData;
+  onChangeSelected: FormsetChange<boolean>;
+  onSetMaxQuantity();
+}
+
+const ItemsCard: React.FC<OrderReturnRefundLinesCardProps> = ({
+  lines,
+  onSetMaxQuantity,
+  onChangeQuantity,
+  onChangeSelected,
+  itemsSelections,
+  itemsQuantities,
+  fulfilmentId,
+  order
+}) => {
+  const classes = useStyles({});
+  const intl = useIntl();
+
+  const handleChangeQuantity = (id: string) => (
+    event: React.ChangeEvent<any>
+  ) => onChangeQuantity(id, parseInt(event.target.value, 10));
+
+  return (
+    <Card>
+      <CardTitle order={order} fulfilmentId={fulfilmentId} />
+      <CardContent className={classes.cartContent}>
+        <MaximalButton onClick={onSetMaxQuantity} />
+      </CardContent>
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableCell>
+              <FormattedMessage
+                defaultMessage="Product"
+                description="table column header"
+              />
+            </TableCell>
+            <TableCell align="right">
+              <FormattedMessage
+                defaultMessage="Price"
+                description="table column header"
+              />
+            </TableCell>
+            <TableCell align="right">
+              <FormattedMessage
+                defaultMessage="Return"
+                description="table column header"
+              />
+            </TableCell>
+            <TableCell align="center">
+              <FormattedMessage
+                defaultMessage="Replace"
+                description="table column header"
+              />
+            </TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {renderCollection(
+            lines,
+            ({
+              quantity,
+              quantityFulfilled,
+              id,
+              thumbnail,
+              unitPrice,
+              productName,
+              variant
+            }) => {
+              const isError = false;
```

Comment:
🤔

Reply:
Stitching is in progress, and the error comes from the backend 😛